ksql-functional-tests/src/test/java/io/confluent/ksql/test/EndToEndEngineTestUtil.java

```diff
@@ -102,26 +106,30 @@ static void writeExpectedTopologyFiles(
     final ObjectWriter objectWriter = new ObjectMapper().writerWithDefaultPrettyPrinter();
 
     testCases.forEach(testCase -> {
-      final KsqlConfig ksqlConfig = new KsqlConfig(baseConfig())
-          .cloneWithPropertyOverwrite(testCase.properties());
-
-      try (final ServiceContext serviceContext = getServiceContext();
-          final KsqlEngine ksqlEngine = getKsqlEngine(serviceContext)) {
-
-        final PersistentQueryMetadata queryMetadata =
-            buildQuery(testCase, serviceContext, ksqlEngine, ksqlConfig);
-        final Map<String, String> configsToPersist
-            = new HashMap<>(ksqlConfig.getAllConfigPropsWithSecretsObfuscated());
-
-        // Ignore the KStreams state directory as its different every time:
-        configsToPersist.remove("ksql.streams.state.dir");
-
-        writeExpectedTopologyFile(
-            testCase.name,
-            queryMetadata,
-            configsToPersist,
-            objectWriter,
-            topologyDir);
+      try {
+        final KsqlConfig ksqlConfig = new KsqlConfig(baseConfig())
+            .cloneWithPropertyOverwrite(testCase.properties());
+
+        try (final ServiceContext serviceContext = getServiceContext();
+            final KsqlEngine ksqlEngine = getKsqlEngine(serviceContext)) {
+
+          final PersistentQueryMetadata queryMetadata =
+              buildQuery(testCase, serviceContext, ksqlEngine, ksqlConfig);
+          final Map<String, String> configsToPersist
+              = new HashMap<>(ksqlConfig.getAllConfigPropsWithSecretsObfuscated());
+
+          // Ignore the KStreams state directory as its different every time:
+          configsToPersist.remove("ksql.streams.state.dir");
+
+          writeExpectedTopologyFile(
+              testCase.name,
+              queryMetadata,
+              configsToPersist,
+              objectWriter,
+              topologyDir);
+        }
+      } catch (final Exception e) {
+        LOG.warn("Ignoring test case {} due to error {}", testCase, e);
```

Comment:
do we actually hit this?

Reply:
Yeah - I hit it for the TOSTRUCT functionality. I meant to remove this and forgot, we should debug it not ignore it.

I'll back this out.